http.go

```diff
@@ -979,24 +979,26 @@ func (req *Request) MayContinue() bool {
 //
 // If maxBodySize > 0 and the body size exceeds maxBodySize,
 // then ErrBodyTooLarge is returned.
-func (req *Request) ContinueReadBody(r *bufio.Reader, maxBodySize int) error {
+func (req *Request) ContinueReadBody(r *bufio.Reader, maxBodySize int, preReadMultipartForm bool) error {
```

Comment:
The only problem I see with this pull request is that this changes the API. Anyone using this function that upgrades fasthttp will have it break their build.

I guess you could make it a `...bool` so it becomes optional and backwards compatible.

Reply:
switched to ...bool (though this seems kind of ugly)